SECP-service/src/main/resources/assets/app/css/app.css

```diff
@@ -1,5 +1,46 @@
+@import url("https://fonts.googleapis.com/css?family=Raleway:400,700");
```

Comment:
do you mind installing these fonts using bower? If something happens to that link, the webpage is screwed.

Reply:
Agreed, best practice is to install all resources needed